recipes/benten/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "benten" %}
+{% set version = "2019.09.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rabix/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: e2fab99296ba80007d15754629483b72df5589c63a34d8b9af5a4873d938a8d7
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37]
```

Looks like a pure Python package, please make this `noarch`
```suggestion
  noarch: python
```